lib/web_ui/lib/src/engine/shadow.dart

```diff
@@ -133,9 +133,24 @@ void applyCssShadow(
   if (shadow == null) {
     element.style.boxShadow = 'none';
   } else {
-    // Multiply by 0.4 to make shadows less aggressive (https://github.com/flutter/flutter/issues/52734)
-    final double alpha = 0.4 * color.alpha / 255;
+    color = toShadowColor(color);
     element.style.boxShadow = '${shadow.offset.dx}px ${shadow.offset.dy}px '
-        '${shadow.blurWidth}px 0px rgba(${color.red}, ${color.green}, ${color.blue}, $alpha)';
+        '${shadow.blurWidth}px 0px rgba(${color.red}, ${color.green}, ${color.blue}, ${color.alpha / 255})';
   }
 }
+
+/// Converts a shadow color specified by the framework to the color that should
+/// actually be applied when rendering the shadow.
+///
+/// Flutter shadows look softer than the color specified by the developer. For
+/// example, it is common to get a solid black for a shadow and see a very soft
+/// shadow. This function softens the color by reducing its alpha by a constant
```

Comment:
Is a linear reduction the best option? Would it make sense for already softer-alpha shadows to reduce by an even smaller factor than 0.3?

Reply:
Happy to change it if there's a concrete use-case. This seems to work fine for the gallery (at least all the places where I could find shadows).

Reply:
Let's chat about a different way to adjust alpha. This seems to fix the immediate issue. We can improve it further in subsequent PRs.